resources/META-INF/studio-contribs.xml

```diff
@@ -16,6 +16,10 @@
     <gradleSyncProvider implementation="io.flutter.android.AndroidStudioGradleSyncProvider" order="first"/>
   </extensions>
 
+  <extensions defaultExtensionNs="io.flutter">
+    <colorPickerProvider implementation="io.flutter.editor.AndroidStudioColorPickerProvider" order="first"/>
```

Comment:
Like with plugin.xml, combine the blocks.

Reply:
done